OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.Teachers.Exceptions
+{
+    public class FailedTeacherServiceException : Xeption
+    {
+        public FailedTeacherServiceException(Exception innerException)
+            : base(message: "Failed teacher service error occured. ",innerException)
+        {
+
```

Comment:
@tharanisantosh remove that extra empty line do CTRL + K + D

Reply:
Removed extra line. Comment resolved.